test/Microsoft.Azure.WebJobs.Extensions.ServiceBus.Tests/MessageToStringConverterTests.cs

```diff
@@ -1,92 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.IO;
-using System.Runtime.Serialization;
-using System.Text;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.ServiceBus.Triggers;
-using Microsoft.Azure.ServiceBus;
-using Xunit;
-using System.Collections.Generic;
-using Microsoft.Azure.ServiceBus.InteropExtensions;
-
-namespace Microsoft.Azure.WebJobs.ServiceBus.UnitTests
-{
-    public class MessageToStringConverterTests
-    {
-        private const string TestString = "This is a test!";
-        private const string TestJson = "{ value: 'This is a test!' }";
-
-        [Theory]
-        [InlineData(ContentTypes.TextPlain, TestString)]
```

Comment:
You should retain these tests, running the data through your new converter logic. That way we're sure we don't have any regressions. These tests were added in the past to guard against regressions.